src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random rand;
```

Comment:
```suggestion
    private Random random;
```
Why abbreviate names? It makes it harder to read